### DISCUSSION TITLE
Zeus CLI issue with getText not updating when generating applets (API 2.0)

### DISCUSSION THREAD

**Discussion by XiaomaiTX** [zepp-health/zeppos-forum#129] [Developer Feedback]
As of now, the getText interface is still API 1.0 in the Empty template I generate when using the Zeus CLI to generate applets (API 2.0), is this a missing bug?
![ScreenCut](https://user-images.githubusercontent.com/59403296/229097945-63d76a27-9fde-41f7-a4b1-ecfd4a4ab35c.png)

**Comment by MyoungXUE** [comment]
Thank you for your feedback, this is indeed a bug, we will update our program template as soon as possible.
